Frontend/ui-common/src/Components/PopUpModal/View.purs

```diff
@@ -103,6 +103,7 @@ view push state =
                 , height WRAP_CONTENT
                 , margin $ MarginVertical 10 10
                 , color Color.black800
+                , gravity CENTER
```

Comment:
@prakharritik can we make it configurable instead of adding gravity here directly.

Reply:
Done